Neos.Utility.Lock/Classes/TYPO3/Flow/Utility/Lock/Lock.php

```diff
@@ -41,17 +36,21 @@ class Lock
     /**
      * @var boolean
      */
-    protected $exclusiveLock = true;
+    protected $exclusiveLock;
 
     /**
      * @param string $subject
-     * @param boolean $exclusiveLock TRUE to, acquire an exclusive (write) lock, FALSE for a shared (read) lock. An exclusive lock ist the default.
+     * @param boolean $exclusiveLock TRUE to, acquire an exclusive (write) lock, FALSE for a shared (read) lock. An exclusive lock is the default.
      */
     public function __construct($subject, $exclusiveLock = true)
```

Comment:
Basically we should remove the API annotation from this class, the factory is the API